pkg/infrared/infrared.go

```diff
@@ -35,20 +37,38 @@ func AddServerConfig(fns ...ServerConfigFunc) ConfigFunc {
 	}
 }
 
+func WithKeepAliveTimeout(d time.Duration) ConfigFunc {
+	return func(cfg *Config) {
+		cfg.KeepAliveTimeout = d
+	}
+}
+
+func DefaultConfig() Config {
```

Comment:
I'm not sure about that. Isn't the whole point to expose your proxy to the internet?

Reply:
I partially agree, for most people that would most likely be the whole point. For some other people its only part of the point. e.g. what if they want to expose it only for one of multiple IP's for their server. 

But this would make them aware of their current setup... I was thinking about this because of databases, which don't allow root users to connect with the database from outside by default...

But I don't have a strong preference for it for the general bindAddr, however i do think we should do this by default for the API endpoint.

Reply:
I totally agree with you on the API part, but I think that having 0.0.0.0 as default binding IP would be helpful for most users. Advanced user normally know what they are doing and can change it manually to what they want/need. Let's keep the default IP at 0.0.0.0, but make sure that the HTTP API will bind to 127.0.0.1.